### ISSUE TITLE
teleport fips rpm fails to install on system with fips_enabled=1

### ISSUE THREAD

**Issue by programmerq** [gravitational/teleport#26264]
### Expected behavior:

The teleport fips rpm should successfully install on a machine with fips mode enabled.

Info about RHEL8 fips mode: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/assembly_installing-a-rhel-8-system-with-fips-mode-enabled_security-hardening

### Current behavior:

The rpm does install on a rhel 8 (or compatible) if it is not in fips mode. If the system has fips mode enabled, it fails.

Bug details:
- Teleport version: all rpm versions appear to be affected. I spot tested several rpms for version 10, 11, 12, and 13 that all experienced this same error.
- Recreation steps: enable fips mode on a RHEL8 system or variant. This can't be done in just a container. The kernel itself must be in fips mode.
- Debug logs

```console
% rpm -ivh teleport-ent-12.3.3-1-fips.x86_64.rpm
warning: teleport-ent-12.3.3-1-fips.x86_64.rpm: Header V4 RSA/SHA512 Signature, key ID 6282c411: NOKEY
Verifying...                          ################################# [100% ]
Preparing...                          ################################# [100% ]
Updating / installing...
   1:teleport-12.3.3-1                ################################# [100% ]
error: unpacking of archive failed on file /lib/systemd/system/teleport.service;646222b8: cpio: Digest mismatch
error: teleport-ent-12.3.3-1-fips.x86_64.rpm: install failed

% cat /proc/sys/crypto/fips_enabled
1
```

### Workaround:

Use the tarball distribution for install. These commands are from [this section](https://goteleport.com/docs/installation/?scope=enterprise#installation-instructions) in the docs. (scope-> enterprise, install, linux, install instructions, tarball tab, "amd64" architecture)

```console
curl https://get.gravitational.com/teleport-ent-v13.0.0-linux-amd64-fips-bin.tar.gz.sha256
curl -O https://cdn.teleport.dev/teleport-ent-v13.0.0-linux-amd64-fips-bin.tar.gz
shasum -a 256 teleport-ent-v13.0.0-linux-amd64-fips-bin.tar.gz
tar -xvf teleport-ent-v13.0.0-linux-amd64-fips-bin.tar.gz
cd teleport-ent
sudo ./install
```

### Workaround 2

Since the rpm digests are the problem, you can tell rpm to skip the digest check:

```console
% rpm -ivh --nodigest --nofiledigest teleport-ent-12.3.3-1-fips.x86_64.rpm
```

**Comment by jcarr444** [comment]
Is there any update as to when this will be patched/released?

**Comment by r0mant** [comment]
This was fixed and released in 13.2.1, 12.4.11 and 11.3.17.
